wiki/Home.md

```diff
@@ -13,29 +13,29 @@ With this package, you can create modified logic trees and use them to calculate
    commands in the terminal:
 
    1. `cd nshm_logic_tree_utilities`
-   2. `mkdir .venv`
-   3. `python -m venv .venv`
+   2. `python -m venv .venv`
 
 3. Activate the new virtual environment by running the following command in the terminal:
    1. `source .venv/bin/activate`
 
-4. Install the required packages by running the following command in the terminal:
-   1. `pip install -r requirements.txt`
+4. Install the required packages by running the following command in the terminal (without changing directories):
+   1. `pip install -e .`
 
 ### Usage
 
 1. To modify the standard NZ NSHM 2022 logic tree and generate hazard curves:
-   1. Open `config.yaml`
-   2. Modify the line containing `output_directory : "/home/arr65/data/nshm/output"` to use your directory
-   3. Run the code with `python run_toshi_hazard_post_script.py`
+
+   1. Go to the `nshm_logic_tree_utilities/scripts` directory (where this `nshm_logic_tree_utilities` directory is inside the first)
+   2. Open `nshm_logic_tree_utilities/run_toshi_hazard_post_config.yaml`
+   3. Modify the line containing `output_directory : "/home/arr65/data/nshm/output"` to use your directory
```

Comment:
More for future, I think it can be better to instead of a file someone has to change to specify the output for a script to have this as a script argument input for an output directory.

Reply:
That's a good suggestion. I'll leave this using a config file for now (just to save time) but if I do more on this project, I might change it to use a script argument input. I'll also keep this in mind for future projects